testdata/book/http_timeout.yml

```
@@ -0,0 +1,20 @@
+desc: Test using HTTP(Timeout)
+runners:
+  req:
+    endpoint: ${HTTPBIN_END_POINT:-https://httpbin.org/}
+    openapi3: https://tryapisproxy.com/spec/httpbin
+    timeout: 1
```

```suggestion
    timeout: 1s
```

Should the conversion from string be done with the time.ParseDuration function?
I think most of the use cases we specify are in seconds.

---

I would like to see you use [duration.Parse](https://github.com/k1LoW/duration) in the same way as in the `loop.interval:` section.
Because I want to align the way to set the time in the runbook.